src/deepak/assignment6/ReadDataFromPropertiesFile.java

```diff
@@ -0,0 +1,238 @@
+		String gender = prop.getProperty("Gender");
+		String livingIn = prop.getProperty("LivingIn");
+		String experience = prop.getProperty("ExperienceInIndustory");
+		String programmingLanguage = prop.getProperty("ProgrammingLanguage");
+		String dreameCompany = prop.getProperty("DreameCompany");
+		
+		driver.findElement(By.xpath("//input[@id='first name']")).sendKeys(firstName);
+		driver.findElement(By.xpath("//input[@id='last name']")).sendKeys(lastName);
+		driver.findElement(By.xpath("//input[@id='E-mail']")).sendKeys(emailAddress);
+		driver.findElement(By.xpath("//input[@id='Company Name']")).sendKeys(companyName);
+		
+		//Step1: Selecting the Gender
+		System.out.println("Step1 : Selecting the Gender");
+		if(gender.equalsIgnoreCase("Male")) {
+			
+			driver.findElement(By.xpath("//input[@id='maleG']")).click();
+		}
+		else
+		{
+			driver.findElement(By.xpath("//input[@id='femaleG']")).click();
+		}
+		
+		//Step2: Selecting value from drop-down
+		System.out.println("Step2: Selecting value from drop-down");
+		WebElement dropDown = driver.findElement(By.xpath("//select[@id='continents']"));
+		
+		Select obj = new Select(dropDown);
+				
+		switch(livingIn) {
+		
+		case "Asia":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+		
+		case "Europe":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+			
+		case "Africa":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+			
+		case "Australia":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+			
+		case "South America":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+			
+		case "North America":
+			
+			obj.selectByVisibleText(livingIn);
+			
+			break;
+			
+		case "Antartica":
+	
+			obj.selectByVisibleText(livingIn);
+	
+			break;
+		}
+		
+		//Step3: Convert the string experience into integer
+		System.out.println("Step3: Convert the string experience into integer");
+		
+		if(experience.equalsIgnoreCase("Student")) {
+			
+			driver.findElement(By.xpath("//input[@id='student']")).click();
+		}
+		
+		else {
+			
+		int yearsOfExp = Integer.parseInt(experience);
+		
+		if(yearsOfExp >=1 && yearsOfExp <3) {
+			driver.findElement(By.xpath("//input[@id='beginner']")).click();			
+		}
+		else if(yearsOfExp >=3 && yearsOfExp < 6) {
+			
+			driver.findElement(By.xpath("//input[@id='entry']")).click();
+		}
+		else if(yearsOfExp >=6 && yearsOfExp < 9) {
+			
+			driver.findElement(By.xpath("//input[@id='intermidiate']")).click();
+		}
+		else if(yearsOfExp >= 10) {
+			
+			driver.findElement(By.xpath("//input[@id='expert']")).click();
+		}
+		
+		}
+		
+		//Step4: Selecting programming Language
+		System.out.println("Step4: Selecting programming Language");
+		
+		List<WebElement> language = driver.findElements(By.xpath("//input[@name='language']"));
+		
+		for(WebElement refVar : language) {
+			
+			if(refVar.isSelected()) {
+				refVar.click();
+			}
+			
+		}
+		
+		
+		String intitialVal = "//input[@id='"; 
```

Comment:
this not efficient code,
take someone's code reference, this is not goodexample of dynamic xpath